integration-web/src/main/resources/integration-web-log-messages.properties

```diff
@@ -33,4 +33,6 @@ integration.web.jwt.pod.url.unauthorized=The provided POD URL {0} is not the sam
   Provide the correct one.  
 integration.web.jwt.pod.token.pair.invalid=The provided application token {0} and Symphony token {1} are invalid. \
   Restart the authentication process again by calling the service 'authenticate'.
-integration.web.jwt.pod.token.jwt.invalid=The provided JWT token {0} is invalid and therefore unauthorized. More information: {1}.
\ No newline at end of file
+integration.web.jwt.pod.token.jwt.invalid=The provided JWT token {0} is invalid and therefore unauthorized. More information: {1}.
```

Comment:
Is the "more information" section a solution for this exception? If so, it could be inserted in the solution list for this